src/resizer/resizer.ts

```diff
@@ -106,11 +106,11 @@ export default class Resizer<C extends IConfig = IConfig> {
     }
 
     public isReverseResizeHandle(el: HTMLElement): boolean {
-        return el && el.classList.contains(this.classNames.reverse);
+        return el && el.classList.contains(this.classNames.reverse!);
```

Comment:
According to the parameter list this can be just

```suggestion
        return el.classList.contains(this.classNames.reverse!);
```

(other occurrence below)